webviz_subsurface/_datainput/opm_init_io.py

```diff
@@ -259,6 +260,58 @@ def create_live_oil(self, raw: EclPropertyTableRawData) -> List[PvxOBase]:
 
         return ret
 
+    def create_dead_oil(self, raw: EclPropertyTableRawData) -> List[PvxOBase]:
+        # Holding raw.num_tables values
+        ret: List[PvxOBase] = []
+
+        column_stride = raw.num_rows * raw.num_tables * raw.num_primary
+        table_stride = raw.num_primary * raw.num_rows
+
+        # pylint: disable=too-many-nested-blocks
+        for index_table in range(0, raw.num_tables):
+            values = []
+
+            # PKey  Inner   C0  C1          C2          C3
+            # Rs    Po      1/B 1/(B*mu)    d(1/B)/dPo  d(1/(B*mu))/dPo
```

Comment:
I don't know the INIT file that well, but is this comment correct? Is the key Rs for a dead oil? Or is it there anyways, and then just 0?